react-client/src/components/Player/VideoJsPlayer.tsx

```diff
@@ -116,7 +115,9 @@ export const VideoJsPlayer = (vpOptions: VideoPlayerOption) => {
         }
       }
       if (vpOptions.media.mime === 'application/x-mpegURL') {
+        try {
         (videoPlayer as any).hlsQualitySelector();
+        } catch { /* continue without hlsQualitySelector */ }
```

Comment:
I don't think it's good to catch and ignore like this. Why is it throwing an error?